docusaurus/docs/Android/04-compose/07-guides/02-customizing-image-and-video-previews.mdx

```diff
@@ -0,0 +1,166 @@
+# Customizing Image and Video Previews
+
+## Introduction
+
+The Compose UI Components chat SDK supports various types of attachments by using [attachment factories](../03-general-customization/01-chat-theme.mdx#streamattachmentfactories). The one used to display image and video attachments is called `MediaAttachmentFactory` and offers a degree of customization.
+
+:::note
+Video Attachment preview thumbnails are a paid feature and as such can be turned off. You can find the pricing [here](https://getstream.io/chat/pricing/) and the information on how to turn them off in the [this](../05-message-components/03-message-list.mdx#image-and-video) section.
```

Comment:
Again, these are technical docs, do we need to mention pricing here?

Reply:
Same as here:

https://github.com/GetStream/stream-chat-android/pull/4373#discussion_r1029655108